examples/snuba-spans/1/basic_span.json

```diff
@@ -0,0 +1,33 @@
+[
+  2,
```

Comment:
This version is setup when an event is sent through the eventstream. Not sure whether spans would be sent via the eventstream or some other interface.

Reply:
removed